awx/ui_next/src/components/LaunchPrompt/useLaunchSteps.js

```diff
@@ -8,51 +9,59 @@ import usePreviewStep from './steps/usePreviewStep';
 export default function useSteps(config, resource, i18n) {
   const [visited, setVisited] = useState({});
   const steps = [
-    useInventoryStep(config, resource, visited, i18n),
-    useCredentialsStep(config, resource, visited, i18n),
-    useOtherPromptsStep(config, resource, visited, i18n),
-    useSurveyStep(config, resource, visited, i18n),
+    useInventoryStep(config, visited, i18n, resource),
+    useCredentialsStep(config, i18n),
+    useOtherPromptsStep(config, i18n),
+    useSurveyStep(config, visited, i18n),
```

Comment:
i like seeing these cleaned up with fewer params… tells me we're likely heading in a good direction 👍🏻